docs/.vuepress/theme/layouts/HomePage/Header.vue

```diff
@@ -0,0 +1,280 @@
+      <PopoverPanel focus class="absolute top-0 inset-x-0 p-2 transition transform origin-top-right lg:hidden">
+        <div class="rounded-lg shadow-lg ring-1 ring-black ring-opacity-5 bg-white divide-y-2 divide-gray-50">
+          <div class="pt-5 pb-6 px-5">
+            <div class="flex items-center justify-between">
+              <div>
+                <img class="h-10 w-auto sm:h-15" src="../../../public/logo-blue.svg" alt="logo" />
+              </div>
+
+              <div class="-mr-2">
+                <PopoverButton
+                  class="
+                    bg-white
+                    rounded-md
+                    p-2
+                    inline-flex
+                    items-center
+                    justify-center
+                    text-gray-400
+                    hover:text-gray-500 hover:bg-gray-100
+                    focus:outline-none focus:ring-2 focus:ring-inset focus:ring-indigo-500
+                  "
+                >
+                  <span class="sr-only">Close menu</span>
+                  <XIcon class="h-6 w-6" aria-hidden="true" />
+                </PopoverButton>
+              </div>
+            </div>
+
+            <div class="mt-6">
+              <nav class="grid gap-y-8 text-center">
+                <a href="#" class="m-0 text-lg font-medium"> Docs </a>
+                <a href="#" class="m-0 text-lg font-medium"> Blog </a>
+                <a href="#" class="m-0 text-lg font-medium"> Community </a>
+                <a href="#"> Slack </a>
+              </nav>
+            </div>
+          </div>
+        </div>
+      </PopoverPanel>
+    </transition>
+  </Popover>
+</template>
+
+<script>
+import { Popover, PopoverButton, PopoverGroup, PopoverPanel } from '@headlessui/vue';
+import {
+  BookmarkAltIcon,
+  CalendarIcon,
+  ChartBarIcon,
+  CursorClickIcon,
+  MenuIcon,
+  PhoneIcon,
+  PlayIcon,
+  RefreshIcon,
+  ShieldCheckIcon,
+  SupportIcon,
+  ViewGridIcon,
+  XIcon,
+} from '@heroicons/vue/outline';
+import { ChevronDownIcon } from '@heroicons/vue/solid';
+
+const solutions = [
+  {
+    name: 'Docs',
+    description: 'Documentation for Kyma.',
+    href: '#',
+  },
+  {
+    name: 'Blog',
+    description: 'Speak directly to your customers in a more meaningful way.',
+    href: '#',
+  },
+  {
+    name: 'Community',
+    description: "Connect with third-party tools that you're already using.",
+    href: '#',
+  },
+  {
+    name: 'Slack',
+    description: 'Build strategic funnels that will drive your customers to convert',
+    href: '#',
+  },
+];
+const callsToAction = [
+  { name: 'Watch Demo', href: '#', icon: PlayIcon },
+  { name: 'Contact Sales', href: '#', icon: PhoneIcon },
+];
+const resources = [];
+const recentPosts = [
+  { id: 1, name: 'Boost your conversion rate', href: '#' },
+  { id: 2, name: 'How to use search engine optimization to drive traffic to your site', href: '#' },
+  { id: 3, name: 'Improve your customer experience', href: '#' },
+];
```

Comment:
As I can see, those arrays aren't used as well. If that's right, please remove them also in the setup method.